### PR TITLE
Added instrumentation and pino for better frontend logging, Updated e…

### PR DESCRIPTION
**Pull request by timwekkenbc**
- [x] Added instrumentation and pino for better frontend logging
- [x] Updated existing logs to use new logger
- [x] Client-side log errors should now be visible on the server logs

### CODE REVIEW

File: logger.ts
Comment (brysonjbest) [inline logger.ts:1]: Was this retained for something?

File: app/api/logError/route.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+interface ErrorRequest {
+  error: string;
+  errorInfo?: string;
+}
+
+export async function POST(req: Request): Promise<Response> {
+  try {
+    const { error, errorInfo }: ErrorRequest = await req.json();
+    // Log the error using Winston
```
Comment (brysonjbest) [inline app/api/logError/route.ts]: Is this intended to be setup with a request to the backend api for logging?

**Review by brysonjbest — APPROVED** [review]
Looks good! Really like how the logging is formatted, and it really cleans it up.

File: logger.ts
Comment (timwekkenbc) [reply logger.ts:1]: Nope, looks like I was a little sloppy this PR.

File: app/api/logError/route.ts (hunk 1 above)
Comment (timwekkenbc) [reply app/api/logError/route.ts]: Nope, just an outdated comment - good catch. Was going to use Winston on the frontend at first before I learned more about its limitations there.

**Pull request merged**